www/src/components/LeaveFeedback.tsx

```diff
@@ -3,7 +3,11 @@ import PropTypes from 'prop-types';
 import { Nav } from '~paragon-react';
 import { useLocation } from '@reach/router';
 
-function LeaveFeedback(props) {
+export interface LeaveFeedbackProps {
+  as?: keyof JSX.IntrinsicElements;
```

Comment:
Is it better to make like this `as?: keyof JSX.IntrinsicElements | React.ElementType`?

Reply:
Done